article/abstract.tex

```diff
@@ -1,13 +1,13 @@
 \section{Abstract}
 
 The value of experimental research articles is inextricably contingent on data analysis results which substantiate their claims.
-However, the intricacy of data analysis procedures and their high reliance on extrinsic tools makes the computation component notoriously fragile.[CITATION NEEDED]
-The inability to re-use procedures due to instability thus endangers their value as a repository of procedural knowledge.
+However, it is well-established that the intricacy of data analysis procedures and their high reliance on extrinsic tools makes the computation component fragile.
+The inability to re-use code due to instability thus limits the lasting value of the work as a repository of procedural knowledge.
 
-It is therefore of crucial importance to approach a higher standard of what constitutes "complete analysis instructions"
+It is therefore of crucial importance to approach a higher standard of what constitutes "complete analysis instructions" to demonstrate the continued reexecutablity and portability of research, especially "keystone results".
```

Comment:
I have concerns about using "complete analysis instructions".
Many systems already do provide nice frameworks for high level instructions to state dependencies and individual steps to achieve results. Usually they are called "workflows" and "pipelines".  There is over 300 of them and one common language to describe them is CWL I recommend to familiarize with https://en.wikipedia.org/wiki/Common_Workflow_Language .

In light of that, as in this paper we are not aiming to provide instructions to humans and what we have a  sandwich full of spaggetti of make/container/bash/python/bash/python/... invocations albeit explicitly coded would not be something someone could have mental capacity to take as instructions for introspection or learning from. 

```suggestion
It is therefore of crucial importance to promote continued reexecutablity and portability of empirical research scholar works, such as scientific articles.
```

Reply:
I think "complete analysis instructions" is an ideal, and should be tool neutral.

Perhaps the only practical way of demonstrating that analysis instructions are complete is to encapsulate all domain knowledge as code, which could not succeed if it missed a step.